website/source/guides/acl.html.markdown

```diff
@@ -347,3 +347,141 @@ Error bootstrapping: Unexpected response code: 500 (Invalid bootstrap reset inde
+```
+
+### Pre-requisites
+- Nomad ACL system bootstrapped.
+- A management token (the bootstrap token can be used, but for production systems it's recommended to
+have a separate token)
+- A set of policies created in Nomad
+- An unsealed Vault server
+
+### Configuration
+Mount the "nomad" secret backend in Vault:
+
+```
+$ vault mount nomad
+Successfully mounted 'nomad' at 'nomad'!
+```
+
+Configure access with the right address and management token:
+
+```
+$ vault write nomad/config/access \
+    address=http://127.0.0.1:4646 \
+    token=adf4238a-882b-9ddc-4a9d-5b6758e4159e
+Success! Data written to: nomad/config/access
+```
+
+Vault secret backends have the concept of roles, configuration unit that group one or more policies
+to a potential identity based on Vault's policy. The name of the role is specified on the path, while
+the mapping to Nomad policies is done by naming them in a comma separated list, for example:
+
+```
+$ vault write nomad/role/role-name policy=policyone,policytwo
+Success! Data written to: nomad/roles/role-name
+```
+
+Alternatively, to create management tokens, or global tokens:
+
+```
+$ vault write nomad/role/role-name token_type=management global=true
+Success! Data written to: nomad/roles/role-name
+```
+
+A Vault policy is required to allow different identities to get tokens associated with a particular
+role:
+
+```
+$ echo 'path "nomad/creds/role-name" {
+  capabilities = ["read"]
+}' | vault policy-write nomad-user-policy -
+Policy 'nomad-user-policy' written.
+```
+
+If you have an existing authentication backend (like LDAP), follow the relevant instructions to create
+a role available on the [Authentication backends page](https://www.vaultproject.io/docs/auth/index.html).
+Otherwise, for testing purposes, a token can be generated associated with the policy:
+
+```
+$ vault token-create -policy=nomad-user-policy
+Key             Value
+---             -----
+token           deedfa83-99b5-34a1-278d-e8fb76809a5b
+token_accessor  fd185371-7d80-8011-4f45-1bb3af2c2733
+token_duration  768h0m0s
+token_renewable true
+token_policies  [nomad-user-policy]
+```
+
+Finally obtain a Nomad Token using the existing Vault Token:
+
+```
+$ vault read nomad/creds/role-name
+Key             Value
+---             -----
+lease_id        nomad/creds/test/6fb22e25-0cd1-b4c9-494e-aba330c317b9
+lease_duration  768h0m0s
+lease_renewable true
+accessor_id     10b8fb49-7024-2126-8683-ab355b581db2
+secret_id       8898d19c-e5b3-35e4-649e-4153d63fbea9
+```
+
+Verify that the token is created correctly in Nomad, referring to it by its accessor:
+
+
+```
+$ nomad acl token info 10b8fb49-7024-2126-8683-ab355b581db2
+Accessor ID  = 10b8fb49-7024-2126-8683-ab355b581db2
+Secret ID    = 8898d19c-e5b3-35e4-649e-4153d63fbea9
+Name         = Vault test root 1507307164169530060
+Type         = management
+Global       = true
+Policies     = n/a
+Create Time  = 2017-10-06 16:26:04.170633207 +0000 UTC
+Create Index = 228
+Modify Index = 228
+```
```

Comment:
Maybe a closing sentence or paragraph to hint as to how to use this? Something like:

> Now any user with access to the Vault policy can obtain a Nomad ACL token. This allows user access to Nomad to be controlled through Vault instead of being managed separately in Nomad.